portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles deleting comments. */
+@WebServlet("/delete-data")
+public class DeleteServlet extends HttpServlet {
+
+  private final DatastoreService datastore;
+
+  public DeleteServlet() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  /** Deletes the comment with the given id parameter */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
You should ideally use the DELETE method for deleting the objects. And use the same servlet as the GET, POST methods to serve it.